formatnumber.go

```diff
@@ -105,8 +139,8 @@ func FormatNumberInt(x int, precision int, thousand string, decimalStr string) s
 	var minus bool
 
 	if x < 0 {
-		if x * -1 < 0 {
-			return FormatNumber(x, precision, thousand, decimalStr)
+		if x*-1 < 0 {
```

Comment:
Could you explain this formatting convention? 😄

Reply:
`go fmt`  does that.